README.md

```diff
@@ -1,74 +1,96 @@
 # roosevelt-logger
 
-Intuitive, attractive logger for Node.js applications based on Winston.
+Intuitive, attractive logger for Node.js applications based on [Winston](https://github.com/winstonjs/winston). This module was built and is maintained by the [Roosevelt web framework team](https://github.com/rooseveltframework/roosevelt), but it can be used independently of Roosevelt as well. 
 
 ## Install
 
-You can install `roosevelt-logger` from NPM:
-
-```
-npm install roosevelt-logger
-```
+First declare `roosevelt-logger` as a dependency in your app.
 
 ## Usage
 
-To use the `roosevelt-logger` simply require the package into your application:
+Require the package into your application and call its constructor:
 
 ```js
 const logger = require('roosevelt-logger')()
+
+logger.log('some info')
+logger.warn('a warning')
+logger.error('an error')
+logger.verbose('noisy log only displayed when verbose is enabled')
 ```
 
-Optionally you can pass the logger a set of configurations. There are default log types that can be enabled and disabled. Additionally custom log types can also be added.
-
-* Default log types: *[Object]*
-  ```json
-  {
-    "info": true,
-    "warnings": true,
-    "verbose": false
-  }
-  ```
-* Custom log types: *[Object]*
-  ```json
-  {
-    "debug": {
-      "enable": true,
-      "type": "error"
+[TODO: show output of above example.]
+
+## Configure logger
+
+Optionally you can pass the logger a set of configs:
+
+- `info` *[Boolean]*: Enable regular logs.
+
+  - Default: `true`.
+
+- `warnings` *[Boolean]*: Enable logging of warnings.
+
+  - Default: `true`.
+
+- `verbose` *[Boolean]*: Enable verbose (noisy) logging.
+
+  - Default: `false`.
+
+- `disable` *[Array of Strings]*: Disable logging in certain environments.
```

Comment:
Can it be clarified further what the strings in this array refer to? I see the example is an environment variable but can other strings be set as well?

Reply:
@mulligan121 can you address this question